docs/PacketFence_News.asciidoc

```diff
@@ -0,0 +1 @@
+../NEWS.asciidoc
```

Comment:
If we add this new guide, we will need to:
- publish it on website
- update description of `all` make target
- update links to `NEWS.asciidoc` in other guides
- reformat `NEWS.asciidoc` according to `PacketFence_Template_Guide.asciidoc`